Retrieve and Rank Integration Tests build failure

In develop branch:

```
/Users/mkistler/Projects/watson-java/wdc-java-sdk/retrieve-and-rank/src/test/java/com/ibm/watson/developer_cloud/retrieve_and_rank/v1/ZipUtilsTest.java:32: error: package com.ibm.watson.developer_cloud.document_conversion.v1.util does not exist
import com.ibm.watson.developer_cloud.document_conversion.v1.util.ConversionUtils;
                                                                 ^
/Users/mkistler/Projects/watson-java/wdc-java-sdk/retrieve-and-rank/src/test/java/com/ibm/watson/developer_cloud/retrieve_and_rank/v1/ZipUtilsTest.java:62: error: cannot find symbol
            ConversionUtils.writeInputStreamToString(new FileInputStream(CONFIG_PATH + "/" + config));
            ^
  symbol:   variable ConversionUtils
  location: class ZipUtilsTest
/Users/mkistler/Projects/watson-java/wdc-java-sdk/retrieve-and-rank/src/test/java/com/ibm/watson/developer_cloud/retrieve_and_rank/v1/ZipUtilsTest.java:78: error: cannot find symbol
        fileMap.put(entry.getName(), ConversionUtils.writeInputStreamToString(input));
                                     ^
  symbol:   variable ConversionUtils
  location: class ZipUtilsTest
3 errors
1 warning
:retrieve-and-rank:compileTestJava FAILED
```

--- a/retrieve-and-rank/build.gradle
+++ b/retrieve-and-rank/build.gradle
@@ -58,6 +58,7 @@ dependencies {
     compile project(':core')
 
     testCompile project(path: ':core', configuration: 'tests')
+    testCompile project(':document-conversion')
 
     signature 'org.codehaus.mojo.signature:java17:1.0@signature'
 }

